docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -0,0 +1,315 @@
+    - Alzheimer's Disease
+  * - BBBM
+    - Blood-Based Biomarkers
+  * - CSF
+    - Cerebrospinal Fluid
+  * - MCI
+    - Mild Cognitive Impairment
+  * - PET
+    - Positron Emission Tomography
+  * - DALYs
+    - Disability-Adjusted Life Years
+  * - CSU
+    - Client Services Unit
+  * - FHS
+    - Future Health Scenarios
+
+1.0 Overview
+++++++++++++
+
+This project leverages IHME's simulation capabilities to quantify health
+and economic impacts associated with early detection and treatment of
+pre-clinical Alzheimer's disease (AD). The simulation evaluates scenarios
+involving blood-based biomarker (BBBM) testing and a hypothetical
+intervention that slows disease progression.
+
+**Basic Goals:**
+
+- Simulate the patient journey from identification through intervention
+  and outcomes
+- Compare health and economic impacts across reference and alternative
+  scenarios in 10 locations
+
+**Funding and collaboration:**
+
+We are designing this simulation in conjunction with IHME's Client
+Services Unit (CSU) with a focus on health and economic impact. Our team
+will focus on simulating the the health impacts of preclinical AD
+testing and the hypothetical intervention, and the Resource Tracking
+team will use our results to estimate the economic impacts. We will be
+using population forecasts from the Future Health Scenarios (FHS) team.
+
+2.0 Modeling Aims and Objectives
++++++++++++++++++++++++++++++++++
+
+The primary goal is to simulate the impact of early detection strategies
+for Alzheimer's disease using blood-based biomarkers and subsequent
+interventions. The simulation tracks simulants through health states
+from age ~30 to 125 years (or death), capturing progression through
+preclinical AD, mild cognitive impairment (MCI) due to AD, and three
+stages of dementia due to Alzheimer's disease.
+
+2.1 Scenarios
+-------------
+
+1. **Reference Scenario:** Present-day conditions with minimal BBBM
+   uptake or disease-modifying therapies, including current
+   cerebrospinal fluid (CSF) and amyloid-positron emission tomography
+   (PET) diagnostic pathways
+2. **Alternative Scenario 1:** Introduction of BBBM testing for at-risk
+   preclinical populations (no intervention)
+3. **Alternative Scenario 2:** BBBM testing plus hypothetical
+   intervention that prevents, delays, or slows disease progression
+
+2.2 General Modeling Strategy
+------------------------------
+
+Based on literature and GBD, we conceive of Alzheimer's disease (AD) as
+comprising a six-stage progression:
+
+  **Susceptible → Preclinical AD → MCI due to AD → Mild AD → Moderate AD
+  → Severe AD**
+
+The last three stages correspond to the three sequelae (mild, moderate,
+severe) of the GBD cause "Alzheimer's disease and other dementias." We
+will have to separate AD out from other dementias in the GBD data, and
+we will need non-GBD data sources to inform our modeling of preclinical
+AD and MCI due to AD. Furthermore, reality may be a bit more complicated
```

Comment:
```suggestion
AD and (possibly) MCI due to AD. Furthermore, reality may be a bit more complicated
```

Reply:
Hmm, seems like now we're pretty sure we want to model MCI, so maybe I should revert this.